packages/cursorless-vscode/src/extension.ts

```diff
@@ -99,6 +100,8 @@ export async function activate(
     fileSystem,
   );
 
+  addCommandRunnerDecorator(new CommandHistory(context, vscodeApi, fileSystem));
```

Comment:
How come this is VSCode-only? I know it's easier to put things in VSCode directory, but I think we should strive to put as much as we can in the engine

Reply:
I had it in the engine originally, but since we changed it to a runner decorator there is no hard engine dependency anymore. It also needs a vscode config so the other option would be to migrate this to our own ide.configuration object?

Reply:
We can do that